en/tidb-operator-overview.md

```diff
@@ -57,15 +68,16 @@ TiDB Operator provides several ways to deploy TiDB clusters in Kubernetes:
 
         You also need to adjust the configuration of the Kubernetes cluster based on [Prerequisites for TiDB in Kubernetes](prerequisites.md) and configure the local PV for your Kubernetes cluster to achieve low latency of local storage for TiKV according to [Local PV Configuration](configure-storage-class.md#local-pv-configuration).
 
-Before deploying TiDB on any of the above two environments, you can always refer to [TiDB Cluster Configuration Document](configure-a-tidb-cluster.md) to customize TiDB configurations.
+Before deploying TiDB on any of the above two environments, you can always refer to [TiDB Cluster Configuration Document](configure-cluster-using-tidbcluster.md) to customize TiDB configurations.
 
 After the deployment is complete, see the following documents to use, operate, and maintain TiDB clusters in Kubernetes:
 
 + [Access the TiDB Cluster](access-tidb.md)
 + [Scale TiDB Cluster](scale-a-tidb-cluster.md)
 + [Upgrade TiDB Cluster](upgrade-a-tidb-cluster.md#upgrade-the-version-of-tidb-cluster)
 + [Change the Configuration of TiDB Cluster](upgrade-a-tidb-cluster.md#change-the-configuration-of-tidb-cluster)
-+ [Backup and Restoration Based on Helm Charts](backup-and-restore-using-helm-charts.md)
++ [Back up a TiDB Cluster](backup-to-s3.md)
++ [Restore a TiDB Cluster](restore-from-s3.md)
```

Comment:
`backup-to-s3.md` and `backup-to-aws-s3-using-br.md` are two separate files. The latter is still under review in https://github.com/pingcap/docs-tidb-operator/pull/45, so the CI check might report deadlink on this one.
```suggestion
+ [Back up a TiDB Cluster](backup-to-aws-s3-using-br.md)
+ [Restore a TiDB Cluster](restore-from-aws-s3-using-br.md)
```